### DISCUSSION TITLE
Relay Robot

### DISCUSSION THREAD

**Discussion by mmmarcusb** [relaycc/relay#105] [Ideas]
TL;DR we can brand Relay Robot with personality, make it a character that would play in our marketing + our own NFT stuff, eventually extend usefulness beyond directory support agent to: 
-Receiver AI helper
-Web3 crawler
-Researching/Arranging/ for execution smart-contract commands/blockchain actions
-Identity data-parse?

______________________
How far are we going to go with the AI integration?  

One can imagine a world where we use it inside Receiver as well, (or across our site) as a generally helpful tool a la Clippy

in which case it could be a branding opportunity where we choose a name with personality and maybe even mascotize it for our Twitter and such (the reddit Snu)

it could also then be extended to NFTs and we can make a value play right from the start where the Relay NFTs are actually like legit.

It's the sort of thing that can bring enormous value but vaporous at the start, but it certainly begins with a vision when choosing the name, and the earlier you decide the more you can do with it.

If we successfully created a useful + entertaining brand icon with a good marketing + NFT strategy in this space it would HUGELY distinguish us from competitors.

It also creates its own opportunities, like the NFT thing which I just realized has powerful tie-ins with both a mascot and the technical uses we have for NFTs internally. We do have to decide on something for our NFTs and people do connect to characters more than just images.
